resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -295,6 +295,8 @@
 @if ($gate)
     <x-app-layout
         :pageTitle="$pageTitle"
+        :pageDescription="$pageDescription ?? null"
+        :pageImage="media_asset('/Images/044930.png')" {{-- hardcoded to use the mature hub icon --}}
```

Comment:
I think this should be hosted locally rather than live on as a media asset.

Reply:
Either that, or it should pull the Adult Hub `Game` entity and fetch the badge from there since it is the ultimate source of truth. Whichever approach we take, the value shouldn't be hardcoded.